### ISSUE TITLE
Bug: Notifications are always disabled when reloading Ferdium

### ISSUE THREAD

**Issue by dmo-odoo** [ferdium/ferdium-app#648]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.0

### What Operating System are you using?

macOS

### Operating System Version

MacOS 12.5

### What arch are you using?

arm64 (including Apple Silicon)

### Expected Behavior

The application remembers whether the notifications were enabled or disabled and set the settings accordingly. Basically, never change the setting without a user interaction.

### Actual Behavior

If I quit Ferdium and reopen it, or if it reloads after a resume, the notifications are disabled.

### Steps to reproduce

Open Ferdium
Enable Notifications
Quit Ferdium
Re-open Ferdium
The notifications are disabled.

### Additional information

I thought there might be a setting to set the default notifications state to disabled and that I might have inadvertantly set it, but I was unable to find such a setting so I'm at a loss.

**Comment by SpecialAro** [comment]
Hello @dmo-odoo ,

I don't have my computer right now but I think there is a setting you should disable (it should be disabled by default, meaning you probably changed it).

When I get the time I can try to take a look 😁

**Comment by meiphoo7-Mae** [comment]
@dmo-odoo Coincidently I saw you question so perhaps I can help. I'm just a regular user like you but since we're part of a community... :wink:  
In the first tab of the settings there's an option about activating a 'don't disturb mode' (it's the last slider). That one requires setting an associated time period or it will use a default period (17.00 - 9.00). I guess it's almost impossible to set that inadvertently but it's the only setting I can find...
Since you mention you can enable notifications I assume you're already aware of the Ctrl + Shift + M shortcut.

**Comment by dmo-odoo** [comment]
Hello, thanks for your replies.

> Hello @dmo-odoo ,
> 
> I don't have my computer right now but I think there is a setting you should disable (it should be disabled by default, meaning you probably changed it).
> 
> When I get the time I can try to take a look 😁

Unfortunately, I was not able to find a setting for that. It's weird honestly, because I would have expected one to be there.

> @dmo-odoo Coincidently I saw you question so perhaps I can help. I'm just a regular user like you but since we're part of a community... 😉 In the first tab of the settings there's an option about activating a 'don't disturb mode' (it's the last slider). That one requires setting an associated time period or it will use a default period (17.00 - 9.00). I guess it's almost impossible to set that inadvertently but it's the only setting I can find... Since you mention you can enable notifications I assume you're already aware of the Ctrl + Shift + M shortcut.

Unfortunately, I do not have scheduled do not disturbed enabled.

I am aware of the shortcut and that is indeed the workaround I'm using right now. However, the problem is to notice that the application has been reloaded and that I have to re-enable notifications again. By the time I realize it, I already missed multiple important notifications. :/

**Comment by dmo-odoo** [comment]
I rebooted my computer today for other reasons and I do not have this issue anymore. By the way, I noticed that the "Do not disturb" mode of Mac OS was somehow enabled. Maybe that was the cause of the Ferdium behavior described here ? I tried turning it back on to reproduce the bug again but even then I could not reproduce the issue. Very weird. Anyway... I do not have this issue anymore so I'm closing it. For other people if you ever face this issue, try disabling do not disturb in Mac OS settings, or reboot your machine and see if it helps.
